### ISSUE TITLE
Cloned device model has single quote in message template

### ISSUE THREAD

**Issue by rickijen** [Azure/device-simulation-dotnet#343]
# Type of issue 

- [x] Bug
- [ ] New feature
- [ ] Enhancement

# Description 

Cloned one of the device models "Chiller" into a new device model "Chiller-Clone". The message template contains single quote and causes problems for applications needed to parse the messages.

# Steps to reproduce 

1. Pick a device model and clone to a new one.
2. Start the simulation with the new device model.
3. From the simulation details window you can see the JSON is mal-formed with single quote message template.

# Expected behavior 

Should be double quoted.

# Current behavior 
![single quote](https://user-images.githubusercontent.com/15071173/53835080-73383b80-3f41-11e9-9392-f046e4ee310e.jpg)

Contains single quote.

# Known workarounds 
Use the advanced option to upload a device model JSON and JavaScript files.

# Possible solution 
Need to fix the bug and generate double quote.

# Context and Environment
* Operating System: ...
* GitHub branch: ...
* .NET Runtime: ...

# Code to reproduce the bug 
```
using Xunit;

public class MyTest
{
    [Fact]
    public void TestToReproduceIssue()
    {
        // Arrange
        ...

        // Act
        ...

        // Assert
        ...
    }
}
```

**Comment by peterfelts** [comment]
Thanks for opening this bug. To confirm, it looks like this isn't breaking the Simulation tool itself. What I think you're reporting is that the message format on cloned device models is messing up consumers of the telemetry. Is this correct?

I noticed that the message format on stock device models (non-cloned, built-in device models) has the message template values escaped.

**Comment by rickijen** [comment]
Correct! Thanks for looking into this.

**Comment by peterfelts** [comment]
I looked in to this today and it looks like the web UI is not using double-quotes for MessageTemplate at all, even when you create a device model from scratch. I pushed a change. 

The fix is very simple, in case you want to clone the web UI repo and copy the change yourself for your current deployment (but that involves creating a new docker image and then going in to the VM (SSH in) and updating /app/docker-compose.yaml to point to your custom image, which is more work than the actual fix...).

I also copied this issue over to the web UI repo (Azure/pcs-simulation-webui#216) so I'll be closing this issue.
